ivy/functional/frontends/torch/tensor.py

```diff
@@ -1188,6 +1188,10 @@ def logdet(self):
         return 2 * torch_frontend.sum(
             torch_frontend.log(torch_frontend.real(torch_frontend.diagonal(chol)))
         )
+    
+    @with_unsupported_dtypes({"1.11.0 and below": ("float16", "bfloat16")}, "torch")
```

Comment:
Here you alright torch not supported bfloat16 and float16 but you must set the version to be " 2.0.1 and below"